web/app/js/components/util/SvgWrappers.jsx

```diff
@@ -41,12 +41,12 @@ export const jaegerIcon = (
+        <path d="M-30.957 138.898c-.803-1.302-2.67-1.275-4.504-1.542-1.2-.177-3.928-.473-4-1.975-.1-1.64 2.314-2.246 3.95-2.592 3.913-.827 7.814-.68 12.526-.68 2.02 0 4.714.127 4.936 1.48.185 1.136-.775 1.947-1.543 2.468-.682.463-2.06.9-2.097 1.914-.068 1.798 3.24 1.47 2.468 3.948-.285.916-1.874 1.44-3.085 1.8-3.84 1.1-8.353 1.477-12.773 1.852-4 .34-9.118 1.105-13.822.432-1.347-.2-3.124-.706-3.208-1.665-.135-1.537 2.4-2.022 3.764-2.223 3.77-.55 7.307-.308 11.415-.493 2.864-.13 5.74-.1 5.984-2.714zm-40.417.556c2.778-.133 6.68.042 9.44.74.722.183 2.095.48 2.098 1.234.004.806-1.808 1.15-2.838 1.42-5.868 1.52-11.88 2-18.387 2.775-1.183.143-2.412.224-3.703.556-.752.194-2.595.742-2.53 1.666.082 1.14 3.03.83 4.258.926 1.58.122 3.58.314 3.702 1.48.208 1.987-3.717 2.075-5.677 2.22-5.4.405-10.042.203-15.056-.432-.977-.123-2.2-.354-3.208-.74-.9-.347-2.134-.868-2.16-1.728-.04-1.285 1.97-1.77 3.085-2.036 4.14-.982 8.652-1.358 12.958-2.283 2.734-.587 5.427-1.633 7.65-2.53 2.428-.98 4.726-2.248 7.22-2.962 1-.283 1.8-.243 3.146-.308zm51.214 4.32c.644-.065 2.397.007 2.406.863.015 1.252-3.937 1.248-4 .062-.047-.742.872-.85 1.624-.925zm-5.06 1.05c.84-.078 2.244-.03 2.284.864.06 1.354-4 1.296-3.95 0 .028-.585.786-.783 1.665-.864zm-6.046 1.046c.37-.028 1.26-.07 1.913.062.237.048 1.207.405 1.234.74.053.635-1.292.874-1.912.925-1.222.103-3.43-.053-3.518-.802-.086-.737 1.566-.872 2.283-.925zm-8.02.618c.97-.13 3.175-.082 3.208.986.048 1.527-4.9 1.535-4.937.062-.018-.795.977-.947 1.73-1.048zm-9.38.112c.823-.056 1.644-.054 2.53.124.43.085 1.756.392 1.8.985.063 1.135-3.138 1.34-4.7 1.235-.945-.062-2.708-.428-2.715-1.172-.006-.63.952-.843 1.234-.926.673-.195 1.048-.2 1.85-.246zm-57.694 2.47c-.295.9-3.22 1.398-4.32.74.574-1 3.033-1.254 4.32-.74zm.68 1.614c-.753 1.114-3.643 1.518-5.06.803.762-1.104 3.64-1.522 5.06-.803zm1.172 1.48c-.805 1.013-3.65 1.514-5.06.742.563-1.034 3.945-1.55 5.06-.742zm5.183.742c-.523 1.247-4.048 1.798-5.245.802.63-1.312 3.904-1.73 5.245-.802zm4.195.246c.787-.062 2.366-.127 3.208.123 1.56.463.332 1.3-.863 1.624-1.23.323-2.566.476-4.135.37-.738-.05-1.98-.247-2.036-.74-.068-.585 1.127-.873 1.542-.988.9-.252 1.47-.305 2.284-.37zm12.342.678c-.07.783-1.12 1.232-2.037 1.543a13.01 13.01 0 0 1-3.085.616c-.685.043-2.04-.004-2.16-.493-.18-.73 1.466-1.3 1.975-1.48 1.053-.378 2.083-.6 3.023-.678.917-.068 1.945-.04 2.284.493zm51.584-119.087c.07.012.034.062 0 .062-4.13 4.635-7.988 9.6-12.834 13.76-.562.483-1.583 1.47-2.345 1.42-.432-.028-.976-.633-1.418-.803-2.6-.993-5.36 1.026-5.245 3.888-.958.293-1.994.4-3.085.493-1.037.1-2.242.352-3.27.247-.76-.078-1.255-.476-2.283-.37-.473.05-.845.358-1.42.432-.9.114-2.1-.193-3.147-.432-6.56-1.5-12.15-4.92-18.573-6.293.792-.483 1.794-.756 2.715-1.1.158-.298-.32-.778-.185-1.05 15.196 1.584 28.443-2.042 39.8-5.8 3.937-1.303 7.775-2.946 11.3-4.442z" />
```

Comment:
Think these got changed by accident when going from rust:1.60 to 1.62

Reply:
thanks, good catch